bower.json

```diff
@@ -1,19 +1,10 @@
 {
   "name": "registries-service",
   "dependencies": {
-    "ember": "2.7.0",
-    "ember-qunit-notifications": "0.1.0",
-    "font-awesome": "4.6.3",
-    "jquery": "2.2.4",
-    "dropzone": "4.3.0",
     "bootstrap": "3.3.7",
```

Comment:
Can this be loaded from NPM instead?

Reply:
We had an issue where ember-osf doesn't have `ember-bootstrap` as a dependency and doesn't build for some reason if it's added as a dependency. So until ember-osf can be upgrading we left bootstrap in bower so that the older version of `ember-bootstrap` (<1.0.0) being used by ember-osf components won't break on registries.